files/dr.conf

```diff
@@ -0,0 +1,6 @@
+[generate_vars]
+site=http://10.35.0.140:8080/ovirt-engine/api
+username=admin@internal
+#password=
+ca=/tmp/ca.pem
+output_file=mapping_vars.yml
```

Comment:
Should probably default to /var/lib/ovirt-ansible-disaster-recovery/mapping.vars.yml, especially if you intend to mainly/only allow editing it using a tool/gui and not manually.

Should probably either refuse changing it if exists, or backup before changing.

Reply:
done